htscodecs/rANS_static4x16pr.c

```diff
@@ -540,7 +542,8 @@ unsigned char *rans_uncompress_O1_4x16(unsigned char *in, unsigned int in_size,
 
     if (!sfb_)
 	return NULL;
-    fb_t fb[256][256];
+    //fb_t fb[256][256];
+    fb_t (*fb)[256] = htscodecs_tls_alloc(256 * sizeof(*fb));
```

Comment:
This allocation also needs to be checked for failure; or it could be combined with the one for `sfb_` as done in `rans_uncompress_O1_32x16()`.